client/landing/gutenboarding/components/domain-picker-button/index.tsx

```diff
@@ -91,11 +95,13 @@ const DomainPickerButton: FunctionComponent< Props > = ( {
 				/>
 			) }
 			{ isDomainPopoverVisible && (
-				<Popover onClose={ handleClose } onFocusOutside={ handleClose }>
+				<Popover onClose={ handleClose } onFocusOutside={ handleClose } focusOnMount="container">
```

Comment:
Setting `focusOnMount="container"` prevents the **Skip for now** button from being focused, as the `Popover` finds the first form element in the content to focus on.

Not done but It is possible to make the domain search input field the field to be initial element to focus on when domain picker appears.